<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuantBTSClass</class>
 <widget class="QMainWindow" name="QuantBTSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BACKTEST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,8,3" columnstretch="2,2,2,2,3">
      <item row="1" column="0" colspan="4">
       <widget class="QChartView" name="k_chart"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="execute_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(76, 255, 88)</string>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="date_end">
        <property name="toolTip">
         <string>End date</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>3</month>
          <day>21</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>12</month>
          <day>27</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1998</year>
          <month>12</month>
          <day>30</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QChartView" name="volume_chart"/>
      </item>
      <item row="0" column="0">
       <widget class="QDateEdit" name="date_start">
        <property name="toolTip">
         <string>Start date</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>1</month>
          <day>7</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>12</month>
          <day>27</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>16</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1998</year>
          <month>12</month>
          <day>30</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="strategy">
        <property name="statusTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>SMA strategy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Buy-Hold strategy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Momentum strategy</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="QTableWidget" name="statistics">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rowCount">
         <number>15</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>48</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="p2.qrc"/>
 </resources>
 <connections/>
</ui>
